controllers/sriov-controller/Makefile

```diff
@@ -0,0 +1,41 @@
+REGISTRY_NAME = 192.168.80.240:4000/ligato
```

Comment:
Remove this before committing, we'll want this in a real docker registry.

Reply:
most likely this Makefile will go away completely since nsm building infra will be used to build controller image.